Sources/Overload/OvMaths/include/OvMaths/FQuaternion.h

```diff
@@ -15,6 +15,11 @@
 
 namespace OvMaths
 {
+	/**
+	* Forward declaration due to circular reference
+	*/
+	struct FMatrix4;
```

Comment:
This isn't necessary as `FMatrix4.` is already included.
It can be either removed, or we can move `#include "OvMaths/FMatrix4.h"` from this file to the `.cpp` file.